corehq/apps/reports/filters/users.py

```diff
@@ -326,7 +324,9 @@ def filter_context(self):
         return context
 
     @classmethod
-    def user_es_query(cls, domain, mobile_user_and_group_slugs):
+    def user_es_query(cls, domain, mobile_user_and_group_slugs, request_user):
+        # The queryset returned by this method is location-safe
+        can_access_all_locations = request_user.has_permission(domain, 'access_all_locations')
```

Comment:
Dont see a use of this variable `can_access_all_locations` except at one place. Might want to just replace that with the expression.

Reply:
Good point, will update